2004-08-18  Jakub Jelinek  <jakub@redhat.com>

	* config/rs6000/crtsavres.asm (.fres): Add .align 2.

--- gcc/config/rs6000/crtsavres.asm.jj	2002-02-19 20:40:41.000000000 +0100
+++ gcc/config/rs6000/crtsavres.asm	2004-08-18 10:58:01.952395671 +0200
@@ -358,6 +358,7 @@ FUNC_END(_savef14)
 
 /* Routines for restoring floating point registers, called by the compiler.  */
 
+	.align 2
 .fres:
 FUNC_START(_restf14)	lfd	14,-144(1)	/* restore fp registers */
 FUNC_START(_restf15)	lfd	15,-136(1)
